regression/test.pl

```diff
@@ -71,7 +71,9 @@ ($$$$$)
 
   $options =~ s/$ign//g if(defined($ign));
 
-  my $output = $input;
+  my $descriptor = $test;
+  $descriptor =~ s/^.*\///;
```

Comment:
I'm going to draw a line here and say: no comments needed: This is a pretty straightforward Regexp string replace. Replaces "^.*\/" with an empty string. Which mans converts path to basename.

This is _the_ standard pattern for doing any kind of string replacement in perl. I know it looks arcane at first glance, but so does putting argument list into a sub. File IO on the other hand...